ops/jenkins/bfd-build-platinum/Jenkinsfile

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env groovy
+
+/**
+ * <p>
+ * This script will be run by Jenkins when building platinum AMI images.
+ * </p>
+ */
+
+// These variables are accessible throughout this file (except inside methods and classes).
+def amiIds
+def platinumAmiId
+def goldAmiId
+def platinumSourceTagAmiId
+
+pipeline {
+    agent {
+        kubernetes {
+            defaultContainer 'bfd-cbc-build'
+            yaml """
+apiVersion: v1
+kind: Pod
+spec:
+  serviceAccount: bfd
+  containers:
+  - name: bfd-cbc-build
+    image: 'public.ecr.aws/c2o1d8s9/bfd-cbc-build:jdk11-mvn3-an29-tfenv-aeaa61fa6'
+    command:
+    - cat
+    tty: true
+    imagePullPolicy: Always
+    resources:
+      requests:
+        memory: '16384Mi'
+        cpu: '8000m'
+      limits:
+        memory: '16384Mi'
+        cpu: '8000m'
+"""
+        }
+    }
+
+    // Run Every day at 8:30 AM Pacific / 3:30 PM UTC
+    triggers {
+        cron('30 15 * * *')
+    }
+
+    stages {
+        stage('Prepare') {
+            steps {
+                script {
+                    // Load the child Jenkinsfiles.
+                    scriptForApps = load('apps/build.groovy')
+                    scriptForDeploys = load('ops/deploy-ccs.groovy')
+                }
+            }
+        }
+
+        stage('Find AMIs') {
+            steps {
+                script {
+                    awsAuth.assumeRole()
+
+                    def amiSourceTagKey = 'amiSourceId'
+                    def goldqueryValue = 'amzn2legacy*'
```

Comment:
Let's re-think the use of these variables, too.

Reply:
Done